qcut/.claude/skills/qcut-toolkit/qcut-slide/references/modification-guide.md

```
@@ -0,0 +1,85 @@
+2. Remove image file and prompt file
+3. **Renumber files**: All subsequent slides decrement NN by 1
+   - `04-slide-conclusion.png` → `03-slide-conclusion.png`
+   - Slugs remain unchanged
+4. Update `outline.md` to remove slide entry
+5. Regenerate PPTX and PDF
+
+## File Naming Convention
+
+Files use meaningful slugs for better readability:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)